controlpanel/settings/common.py

```diff
@@ -248,6 +249,19 @@
 # Whitelist values for the HTTP Host header, to prevent certain attacks
 ALLOWED_HOSTS = [host for host in os.environ.get("ALLOWED_HOSTS", "").split() if host]
 
+
+# set this before adding the IP address below
+# TODO We may be able to set this in terraform instead, we should check this
+QUICKSIGHT_DOMAINS = []
```

Comment:
I think this is going to cause problems in dev/prod, as the Control Panel helm chart sets `ALLOWED_HOSTS` to a [wildcard](https://github.com/ministryofjustice/analytics-platform-helm-charts/blob/dcf007bfbc36b26277295a34ceabbbf99137ca34/charts/cpanel/templates/env-configmap.yaml#L12) (which is not good in any case)

So left as it is, I'd expect Quicksight to fail to embed when deployed. Instead, we need to update the ALLOWED_HOSTS to explicitly include the the prod and dev domains: `controlpanel.services.analytical-platform.service.justice.gov.uk` and `controlpanel.services.dev.analytical-platform.service.justice.gov.uk`.

We have a few options for doing this, but I think the best one it to add `ALLOWED_HOSTS` as an env var in the flux repo. Not sure if this would then take precendence over the one set in the helm chart, but we could try it. Or delete the var in the helm chart, and publish a new version.